web/src/compose.js

```diff
@@ -358,11 +353,19 @@ function schedule_message_to_custom_date() {
         req_type = message_type;
     }
 
+    let message_to;
+    if (message_type === "private") {
+        const recipient = compose_state.private_message_recipient_emails();
+        message_to = util.extract_pm_recipients(recipient);
```

Comment:
This seems like it does not match the previous behavior -- we previously ended up with a user ID array. See https://github.com/zulip/zulip/pull/35921/commits/d10cccbdb85928e44f9b92e72cad6920f177826b#r2320358287, but note in particular we really don't care about scheduled messages for Zephyr.

Reply:
You're right that I missed the Zephyr case, but this does match the default case above. But if we don't care about the Zephyr case, does that mean this is fine then?

Reply:
No, this matches the Zephyr case -- you're sending emails, not user IDs, when in fact we want to always be using user IDs, because only Zephyr's weird corner case needed emails. (I don't think that code path works anymore, since AFAIK we removed the ability to store email addresses without a user ID in compose pills last year).